### PR TITLE
Kayla

### PR DESCRIPTION
**Pull request by kaykuhl**
Changes are live on Azure

### CODE REVIEW

**Comment by kaykuhl** [comment]
Updates 1/5/2021: 
Added logic to remove Submit Button after upload is complete / added Reset Button
Added error for document type that doesn't match
CSV File Generated for Errors / Added Button for Download
Errors that don't have a mapping file line associated like "Resume2.pdf has no match in the mapping file" will show up here as a line.  Errors that have a mapping file line will have that additional information (file_name, system_id, etc.) filled in for easier resubmittal after errors are fixed. 
Updated CSS
Updated Instructions
Added additional information and reorganization.

File: .sdfcli.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (JonnyBoy333) [inline .sdfcli.json:1]: Did this file get added by mistake? I don't think there should be an SDF file in this project.

File: package.json [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
     "license": "ISC",
     "dependencies": {
-        "axios": "^0.18.1",
-        "basic-auth": "^2.0.0",
-        "express": "^4.16.3",
-        "express-basic-auth": "^1.1.7",
-        "multer": "^1.3.0",
-        "papaparse": "^5.2.0"
+        "axios": "^0.21.1",
+        "dotenv": "^8.2.0",
```
Comment (JonnyBoy333) [inline package.json:14]: Dotenv should be a dev dependency since it's not used on the production server

**Review by JonnyBoy333 — CHANGES_REQUESTED** [review]

File: app.js [hunk 3]
```diff
@@ -1,46 +1,39 @@
...
-//     res.setHeader('WWW-Authenticate', 'Basic realm="Node"')
-//     res.end('Incorrect username or password.')
-//   } else {
-//     next();
-//   }
-// });
-
+const app = require('express')();
+const basicAuth = require('express-basic-auth');
+require('dotenv').config();
```
Comment (JonnyBoy333) [inline app.js:13]: dotenv should only be required in the development environment. Historically I've used the preload option be adding to the node run command. You should also make sure you're environmental variables are added to the Application Settings in the Azure app.

**Review by JonnyBoy333 — CHANGES_REQUESTED** [review]

File: .sdfcli.json (hunk 1 above)
Comment (kaykuhl) [reply .sdfcli.json:1]: Yes, it was a mistake that it was added.

**Comment by kaykuhl** [comment]
Moved dotenv to dev dependencies and using preload/Azure env for password variables. Pushed to production and authentication functionality still working.

**Review by JonnyBoy333 — APPROVED** [review]

**Pull request merged**